modules/greenbidsRtdProvider.js

```diff
@@ -85,6 +85,7 @@ function processSuccessResponse(response, timeoutId, reqBidsConfigObj, greenbids
 
 function updateAdUnitsBasedOnResponse(adUnits, responseAdUnits, greenbidsId) {
   const isFilteringForced = getParameterByName('greenbids_force_filtering');
+  const isFilteringDisabled = getParameterByName('greenbids_disabled_filtering');
```

Comment:
Stay consistent with the other parameter name.
```suggestion
  const isFilteringDisabled = getParameterByName('greenbids_disable_filtering');
```